README.md

```diff
@@ -24,7 +26,16 @@ In the European Union, this extension's fields correspond to [eForms BG-704 (Rew
       ],
       "hasPrizes": true,
       "prizes": {
-        "description": "The winner(s), as well as each unsuccessful competitor who has provided services which meet the program, will receive, subject to the decision of the contracting authority, a prize of EUR 16,500.00 (VAT free)."
+        "description": "The winner(s), as well as each unsuccessful competitor who has provided services which meet the program, will receive, subject to the decision of the contracting authority, a prize of EUR 16,500.00 (VAT free).",
```

Comment:
good catch, those values should be the same but I don't think the sentence needs changed. My reading is that it's still a single prize even if multiple parties are awarded it as long as the value and prize reason/identity is the same. A bit like how 2 people can both win Gold medals in the 100m sprint if they both run exactly the same time. I'd consider that one prize awarded to 2 people rather than 2 separate prizes. So the `id` would be "Gold" and both winners get it if that makes sense?

Reply:
Makes sense!